backends/gdx-backend-headless/src/com/badlogic/gdx/backends/headless/HeadlessPreferences.java

```diff
@@ -34,18 +34,34 @@
 
 public class HeadlessPreferences implements Preferences {
 	private final Properties properties = new Properties();
-	private final FileHandle file;
+	private FileHandle file;

```

Comment:
Also here, again not sure if this `FileHandle` needs to be a member when it seems to be only used in a constructor.

Reply:
For some reason, I didn't see that there was a reference already to `file`, and only its `final` status changed. So these should work.